src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(/images/star.svg);
+  }
+  
+  &__star:nth-child(4n),
+  &__star:nth-last-child(-n + 4) {
+    background-image: url(/images/star-active.svg);
+  }
```

Comment:
shouldn't be hardcoded, use solution from this [task](https://github.com/mate-academy/layout_stars)